CodeEditModules/Modules/WelcomeModule/src/RecentProjectsView.swift

```diff
@@ -74,9 +74,10 @@ public struct RecentProjectsView: View {
         )
     }
 
-    /// Update recent projects.
+    /// Update recent projects, and remove ones that no longer exist
     func updateRecentProjects() {
-        recentProjectPaths = UserDefaults.standard.array(forKey: "recentProjectPaths") as? [String] ?? []
+        recentProjectPaths = ( UserDefaults.standard.array(forKey: "recentProjectPaths") as? [String] ?? [] )
```

Comment:
Shouldn't we also be removing the path entry from `UserDefaults` as well?

Reply:
ah, thought that it was already being done but I was looking at `deleteFromRecent` instead :P
Its fixed now.